docs/user-guide/map-views.md

```diff
@@ -0,0 +1,67 @@
+# Map Views
+
+The **Map Views** is a [MapStore](https://mapstore.geosolutionsgroup.com/mapstore/#/) tool useful to configure multiple views and switch between them with navigation tool.
+
+!!! note
+    The *Map views* works in both 2D and 3D modes, but the 3D mode has advanced options including the *Mask*, the *Globe Transluncency* and the *Clipping* (as we will see in the next paragraph).
+
+## Add new view
+
+Once the user opens a map, in [3D Mode](navigation-toolbar.md#3d-navigation), the *Map Views* tools can be open through the <img src="../img/button/map-views-button.jpg" class="ms-docbutton" /> button available in the [Side Toolbar](mapstore-toolbars.md#side-toolbar).
+
+<img src="../img/map-views/map-views-panel.jpg" class="ms-docimage"/>
+
+To create a new view, the user can, using the compass located in the upper right corner of the map, highlight a specific area on the map, enable the layers, on TOC, to be displayed in the map and finally click on the <img src="../img/button/+++.jpg" class="ms-docbutton" />. This view will be visible in the *Views* list by clicking the <img src="../img/button/timeline-layers-list-button.jpg" class="ms-docbutton" /> button.
+
+<img src="../img/map-views/add-view.gif" class="ms-docimage"/>
+
+[MapStore](https://mapstore.geosolutionsgroup.com/mapstore/#/) allows to customize and edit the new view by opening the Edit panel through the <img src="../img/button/editing-button.jpg" class="ms-docbutton" /> button. Here the user is allowed to:
+
+* Add a text, images, videos or hyperlinks by clicking **Description** section. The *description* is visible during the [Views Navigation](map-views.md#3d-views-navigations). Take a look at the following example.
+
+<img src="../img/map-views/description.jpg" class="ms-docimage"/>
+
+* Change the longitude, latitude and height of the *Camera Position* and *Center Position* by clicking on **Position** section.
+
+<img src="../img/map-views/position.jpg" class="ms-docimage"/>
+
+* Modify the duration of the animation and enable/disable the transition during the [Views Navigation](map-views.md#3d-views-navigations) by clicking on **Animation** section.
+
+<img src="../img/map-views/animation.jpg" class="ms-docimage"/>
+
+* Select a WFS or Vector layer available in the map to create a mask on on all the 3D tiles visible in the map by clicking **Mask** section. The WFS or Vector layer is added to the map by the user and then used in a transparent way to create the clipping area.
+
+<img src="../img/map-views/mask-panel.jpg" class="ms-docimage"/>
+
+If the user enables the **Inverse** option, a hole is created inside the 3D Tile layers. This hole can be filled by only 3D objects (3D Tiles cannot be used to fill these holes because all 3D tiles will be masked).
```

Comment:
"... This hole can be filled by only 3D objects (3D Tiles cannot be used to fill these holes because all 3D tiles will be masked)."

@allyoucanmap I don't think that sentence is correct. Can you confirm?